core/layouts/rolph/rolph.info

```diff
@@ -0,0 +1,23 @@
+name = Rolph
+version = BACKDROP_VERSION
+backdrop = 1.x
+type = layout
+
+; Specify regions for this layout.
+regions[header]           = Header
+regions[top]              = Top
+regions[quarter1]         = First Column
+regions[quarter2]         = Second Column
+regions[quarter3]         = Third Column
+regions[quarter4]         = Fourth Column
```

Comment:
If we stick with current naming conventions, these would become "quarter_first" through "quarter_fourth".